src/clr-base/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@clr/base",
+    "version": "2.0.2",
+    "description": "Custom Elements for Clarity",
+    "homepage": "https://clarity.design",
+    "main": "index.js",
+    "types": "index.d.ts",
+    "repository": {
+        "type": "git",
+        "url": "ssh://git@git.eng.vmware.com/clarity.git"
+    },
+    "peerDependencies": {
+        "@webcomponents/shadycss": "1.9.1",
+        "@webcomponents/webcomponentsjs": "2.2.10",
+        "lit-element": "2.2.0"
```

Comment:
question: will `@clr/base` need to keep up/release major versions with lit-element versions, in a similar way that `@clr/angular` matches `@angular/*`?

Reply:
`@clr-base` does not have to keep in sync with lit-element. We define what version we want to use. You can have multiple versions in the same app, it gets bundled into the module and webpack can dedupe versions at the application level. Ideally though best practices we should stay on latest the same way we do with Angular.